package-lock.json

```diff
@@ -19317,7 +19332,7 @@
         "rfdc": {
             "version": "1.1.4",
             "resolved": "https://registry.npmjs.org/rfdc/-/rfdc-1.1.4.tgz",
-            "integrity": "sha512-5C9HXdzK8EAqN7JDif30jqsBzavB7wLpaubisuQIGHWf2gUXSpzy6ArX/+Da8RjFpagWsCn+pIgxTMAmKw9Zug==",
+            "integrity": "sha1-unLME2egzNnPgahws7WL060H+MI=",
```

Comment:
We have the same issue here

Reply:
Make sure you resolve these integrity issues

Reply:
resolved